### DISCUSSION TITLE
Few improvements

### DISCUSSION THREAD

**Discussion by grelix** [elementary/triage#537] [Ideas]
Hello, 

1. during the installation of Elementary OS, there is a small issue at the language selection stage - some languages (like Polish) are written in lowercase, while others are written in uppercase. It seems that the sorting method is case-sensitive, causing some languages to be out of order. I have several friends who chose a different operating system because they couldn't find the Polish language on the list.
2. During installation, the language selection actually occurs twice - once before the installation and again at the first startup. This is somewhat confusing, and I believe that after the first startup, the system should remember the language choice made during the installation stage.
3. During the installation of the system, there should be an option to enable the firewall. Both I and several Linux distro reviewers have noted that a system focused on privacy and security, and used by new users, does not provide this capability by default. It's important to remember that new users may not be aware that they need to enable the firewall separately.
4. Please consider integrating a solution for synchronization with phones. It would be great to have the ability to create an ecosystem with Android phones right out of the box, without the need to install additional software.
5. I, along with several friends who have tried Elementary OS, have noticed the lack of a dedicated notepad application. I currently use the default Code application as a makeshift notepad, but that's not its intended purpose. In my opinion, a notepad application should be included alongside the Code application.
6. In my opinion, the kernel version should be upgraded. There are some conflicts with newer hardware, including issues with the AMD GPU for the Ryzen 7520U processor, which becoming popular. Using the current kernel causes the system to freeze and occasional display problems. After manually upgrading the kernel to version 6.9.3, all issues were resolved, and the system became significantly faster and more responsive.

**Comment by danirabbit** [comment]
Hey thanks for opening this discussion! Issue reports need be tracked as separate reports with one report per issue. It's not possible for the team to assign and track group issue reports like this. Please see [the contributor guide](https://docs.elementary.io/contributor-guide/feedback/reporting-issues) for more info

Closing since this isn't actionable as reported
